### PR TITLE
Describe ECALL/EBREAK operation in privileged architecture

### PR DESCRIPTION
**Pull request by aswaterman**
#54

### CODE REVIEW

File: src/machine.tex [hunk 1]
```diff
@@ -1830,6 +1830,54 @@ \subsection{Machine Bad Bits ({\tt mtval}) Register}
...
+\multicolumn{1}{c|}{opcode} \\
+\hline
+12 & 5 & 3 & 5 & 7 \\
+ECALL   & 0 & PRIV & 0 & SYSTEM \\
+EBREAK  & 0 & PRIV & 0 & SYSTEM \\
+\end{tabular}
+\end{center}
+
+The ECALL instruction is used to make a request to the supporting execution
+environment.  When executed in U-mode, S-mode, or M-mode, it generates an
```
Comment (sorear) [inline src/machine.tex]: "Generates an environment call" is a little bit vague.  Would something more like «generates an exception of type "environment call from U-mode", "environment call from S-mode", or "environment call from M-mode", respectively» work?

Also, would it make sense to add a commentary to the user spec saying that the runtime behavior is documented elsewhere?

Comment (sorear) [inline src/machine.tex]: On that note we don't seem to have anything in the privilege spec that spells out the sequence of events for a trap in one place, although §3.1.13 (medeleg) covers most of them.

Comment (aswaterman) [inline src/machine.tex]: Thanks, that was an editing error.

Comment (aswaterman) [inline src/machine.tex]: The user spec already implies the runtime behavior is defined elsewhere

> The ABI for the system will define how parameters for the environment request...

and indeed I think it's best to leave it described as an ABI thing.

**Pull request merged**